### PR TITLE
Ep2 us2 levin visuals for opening sequence

### PR DESCRIPTION
**Pull request by levinthiel**
Creation of the opening cinematic artwork

### CODE REVIEW

**Comment by levinthiel** [comment]
@StephMode  pls have a lookie <3

File: app/OpeningCinematicPage/page.js [hunk 1]
```diff
@@ -9,14 +9,14 @@ export default function OpeningCinematicPage() {
         <Wrapper>
             <MovieDisplay/>
             <TextBox>
-              <NextTxtButton> &#11208; </NextTxtButton>
+              <NextTxtButton>  </NextTxtButton>
```
Comment (StephMode) [inline app/OpeningCinematicPage/page.js:12]: Let's change the name of this component.

I see room for improvement in the following aspects:
1) We should refrain from omitting vocals in names of functions, variables and so on (I've read about this in quite a few blogs now).
2) The name itself could be more descriptive.

My suggestions:
on 1) make it at least `NextTextButton`, or consider changing the name alltogether.

on 2)
Improve name of the component by:
- including an action verb, like: `ContinueTextButton`
- using a name to reflect component structure: `TextBoxButton` (could be the most straight-forward as we most likely won't have any more buttons in the text box).

File: app/OpeningCinematicPage/page.js [hunk 2]
```diff
@@ -64,18 +64,18 @@ const TextBox = styled.div`
...
   @media only screen and (max-width: 400px) {
-    background: url(/ui375.png) center no-repeat;
+    background: url(/ui-375.png) center no-repeat;
     background-size: cover;
   }
 `;
 const NextTxtButton = styled.button`
   border: none;
-  background: #639B85;
+  background: url(/arrow.png) no-repeat;
```
Comment (StephMode) [inline app/OpeningCinematicPage/page.js:73]: Nice implementation :)

**Review by StephMode — CHANGES_REQUESTED** [review]
Great work! I love how you improved the UI and the artworks you created are wonderful.
There's just a small issue with the naming of one component I'd like to have fixed.

File: app/OpeningCinematicPage/page.js (hunk 1 above)
Comment (levinthiel) [reply app/OpeningCinematicPage/page.js]: Good Idea, I changed it ;)

Comment (levinthiel) [reply app/OpeningCinematicPage/page.js]: please review ;)

**Review by StephMode — APPROVED** [review]
Looks good to me now. Let's merge :)

**Pull request merged**